broker/hcaxlsbroker.py

```diff
@@ -72,7 +72,7 @@ def createSubmission(self):
     # this doesn't support <level1>.<level2>.<level3>
```

Comment:
Can you put a comment over the top of this function describing the python types of the parameters and a small description/example of what the function does?

Reply:
will do